hack/local-up-cluster.sh

```diff
@@ -27,7 +27,6 @@ DOCKER_OPTS=${DOCKER_OPTS:-""}
 export DOCKER=(docker "${DOCKER_OPTS[@]}")
 DOCKER_ROOT=${DOCKER_ROOT:-""}
 ALLOW_PRIVILEGED=${ALLOW_PRIVILEGED:-""}
-DENY_SECURITY_CONTEXT_ADMISSION=${DENY_SECURITY_CONTEXT_ADMISSION:-""}
```

Comment:
This line ensures that `DENY_SECURITY_CONTEXT_ADMISSION` is set. Shell code in Kubernetes is picky about that and will fail when expanding an unset variable.

Removing it looks suspicious.

Reply:
So this is mostly garbage collection since this won't be used anymore, that variable was added explicitly for the `SecurityContextDeny` admission plugin and will be useless now. See history:

- https://github.com/kubernetes/kubernetes/commit/9238f38400ccf513fda9c728e1d41841df7e2c77
- https://github.com/kubernetes/kubernetes/commit/3d4b76d7226452aa785d2c028c6e388bb539aace

Reply:
And on addition, to me `DENY_SECURITY_CONTEXT_ADMISSION` is not used anyway in this shell script anymore after my patch, so not expanded?

Reply:
You are right, I misread the patch.